vue/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vue-text-mask",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
I'm pretty new to Git and versioning, so not too sure what I need to do with this one